Remove "...create_host_config deprecated" warning

nrhodes opened the pull request:
Remove following warning:
WARNING:py.warnings:/Library/Python/2.7/site-packages/docker/utils/utils.py:486:
UserWarning: docker.utils.create_host_config() is deprecated. Please use
Client.create_host_config() instead.

nrhodes committed:
Remove "...create_host_config deprecated" warning

Remove following warning:
WARNING:py.warnings:/Library/Python/2.7/site-packages/docker/utils/utils.py:486:
UserWarning: docker.utils.create_host_config() is deprecated. Please use
Client.create_host_config() instead.

saeta reviewed (approved):
Looks great to me! Thanks for improving this. 

One quick question, do you know when they added `docker.create_host_config`? (We should update `setup.py`'s dependency requirements to ensure old docker library dependencies get updated if needed.)

nrhodes commented:
Looks as though that got added in docker-py 1.4.0
http://docker-py.readthedocs.io/en/1.4.0/change_log/.

Neil

On Fri, Oct 14, 2016 at 1:10 PM, Brennan Saeta notifications@github.com
wrote:

> _@saeta_ approved this pull request.
> 
> Looks great to me! Thanks for improving this.
> 
> One quick question, do you know when they added docker.create_host_config?
> (We should update setup.py's dependency requirements to ensure old docker
> library dependencies get updated if needed.)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/coursera/courseraprogramming/pull/69#pullrequestreview-4336309,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AF_GOdba3GgdLy2L8MFr0oCsizMOEsvmks5qz-G5gaJpZM4KXZL6
> .

saeta commented:
Excellent. If you modify https://github.com/coursera/courseraprogramming/blob/master/setup.py#L32 such that it reads 1.4.0 for the minimum version required, I'll merge this in!

nrhodes committed:
Update version of docker-py

Updated to 1.4.0 since that's when create_host_config moved from
docker.utils to Client.

nrhodes commented:
Done.

On Fri, Oct 14, 2016 at 1:19 PM, Brennan Saeta notifications@github.com
wrote:

> Excellent. If you modify https://github.com/coursera/
> courseraprogramming/blob/master/setup.py#L32 such that it reads 1.4.0 for
> the minimum version required, I'll merge this in!
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/coursera/courseraprogramming/pull/69#issuecomment-253909090,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AF_GOfU_F4X7JsAFYDctqysIRiYTDXlSks5qz-PEgaJpZM4KXZL6
> .

saeta commented:
Following up here, version 0.17.0 has been released and published.
